src/components/academies/tests/AcademyDetailPage.test.jsx

```diff
@@ -108,6 +91,12 @@ describe('<AcademyDetailPage />', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     useEnterpriseCustomer.mockReturnValue({ data: mockEnterpriseCustomer });
+    useAlgoliaSearch.mockReturnValue({
+      searchClient: { search: jest.fn().mockResolvedValue(ALOGLIA_MOCK_DATA) },
+      searchIndex: { search: jest.fn().mockResolvedValue(ALOGLIA_MOCK_DATA) },
+      shouldUseSecuredAlgoliaApiKey: false,
+      hasCatalogUuidToCatalogQueryUuidMapping: false,
```

Comment:
nit: this hook returns the `catalogUuidsToCatalogQueryUuids`, but not `hasCatalogUuidToCatalogQueryUuidMapping`.